config/quickLookIsr.py

```diff
@@ -3,3 +3,4 @@
 config.doSaturationInterpolation = True
 config.overscan.fitType = "MEDIAN_PER_ROW"
 config.overscan.doParallelOverscan = True
+# config.brighterFatterMaxIter = 2  # Uncomment this to remove test warning
```

Comment:
I think this should go back in - that's an acceptable warning to have.

Reply:
ok